htdocs/compta/bank/card.php

```diff
@@ -65,7 +65,7 @@
 $extrafields->fetch_name_optionals_label($object->table_element);
 
 // Initialize technical object to manage hooks of page. Note that conf->hooks_modules contains array of hook context
-$hookmanager->initHooks(array('bankcard', 'globalcard'));
+$hookmanager->initHooks(array('bankaccountcard', 'globalcard'));
```

Comment:
To remove hardcoded value, context for hook on object card should now be
`$hookmanager->initHooks(array($object->element.'card', 'globalcard'));`